ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -15,13 +15,17 @@ import { UnitOptions, LegendOptions } from '@perses-dev/components';
 import { ThresholdOptions } from '@perses-dev/core';
 import { OptionsEditorProps } from '@perses-dev/plugin-system';
 
+export interface TimeSeriesChart {
+  kind: 'TimeSeriesChart';
+  spec: TimeSeriesChartOptions;
+}
+
 /**
  * The Options object supported by the TimeSeriesChartPanel plugin.
  */
 export interface TimeSeriesChartOptions {
   legend?: LegendOptions;
   y_axis?: YAxisOptions;
-  unit?: UnitOptions;
```

Comment:
I think this is legacy, since we configure `unit` inside of `y_axis` at the moment

Reply:
Gooooood call